kong/clustering/rpc/manager.lua

```diff
@@ -201,6 +201,11 @@ function _M:_handle_meta_call(c)
   local capabilities_list = info.rpc_capabilities
   local node_id = info.kong_node_id
 
+  -- we already set this dp node
+  if self.client_capabilities[node_id] then
```

Comment:
I would suggest not. If a DP is updated with a new capability, it won't be updated here

Reply:
Please refer to `_remove_socket()`, if dp disconnects, cp will remove all connections and set it to nil. When dp starts again, cp should only set once for this dp.